resources/translations/fi.edn

```diff
@@ -286,6 +286,6 @@
            :open-search "Avaa haku"
            :search-parameters "Hakutermit"}
   :unauthorized-page {:unauthorized "Ei käyttöoikeuksia"
-                      :you-are-unauthorized "Sinulla ei ole käyttöoikeuksia tähän sivuun."}
+                      :you-are-unauthorized "Ei sivun käyttöoikeuksia."}
   :forbidden-page {:forbidden "Kielletty"
-                   :you-are-forbidden "Sinulla ei ole pääsyä tähän sivuun."}}}
+                   :you-are-forbidden "Sivun käyttö kielletty."}}}
```

Comment:
This should be matched to the heading. It can be shorter and the main explanation should be longer. See also English.

Reply:
Isn't this already matched? "Kielletty" vs. "Sivun käyttö kielletty."